### PR TITLE
revert field number change

### PR DESCRIPTION
**Pull request by katrogan**
Signed-off-by: Katrina Rogan <katroganGH@gmail.com>

# TL;DR
Reverts field number that was accidentally changed in https://github.com/flyteorg/flyteidl/commit/e46cb9fd7163a2426e1db63dea6f7326cb86de6a#diff-724a7e8bb673471da21d8cb4a8ef634198bc10cc7b61965a920673c2b7408a54L238

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Comment by katrogan** [comment]
closing in favor of https://github.com/flyteorg/flyteidl/pull/127
